### DISCUSSION TITLE
Welcome to gatl Discussions!

### DISCUSSION THREAD

**Discussion by laffernandes** [laffernandes/gatl#25] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by CaseySanchez** [comment]
This is perfect, thank you Dr. Fernandes.

My name is Casey Sanchez and I am an undergrad of Electrical Engineering at the University of Central Florida, where I am a member of IEEE UCF and am currently working on the [Autonomous Surface Disinfection Robot to combat SARS-CoV-2](https://github.com/IEEE-UCF/ASDR) project.

My next goal is to create a bipedal humanoid robot using GATL for the underlying mathematics. To meet this end, I have thus far created a [transform class](https://gist.github.com/Tannz0rz/a475eb8872cd12e17b82b579b277d46b) to implement complex transform hierarchies, and am currently working on implementing the FABRIK (Forward And Backward Reaching Inverse Kinematics) algorithm with joint constraints. Once this is completed, I would then need to research gait optimization methods to teach the baby to walk independently.

I'm still in the process of learning GA myself, so I'm hoping to learn more by doing. Cheers!
